gameSource/languages/Polish.txt

```diff
@@ -0,0 +1,276 @@
+writeFailed "BŁĄD:  FOLDER Z GRĄ JEST W TRYBIE TYLKO OD ODCZYTU"
+
+
+measuringFPS "MIERZENIE MAKSYMALNYCH FPS"
+
+vsyncWarning "OSTRZERZENIE: SYNCHRONIZACJA PIONOWA WYŁĄCZONA"
+vsyncWarning2 "(PRAWDOPODOBNIE BĘDZIE TO MIEĆ NEGATYWNY WPŁYW NA WRAŻENIA WIZUALNE)"
+
+vsyncWarning3 "MOŻESZ SPRÓBOWAĆ NAPRAWIĆ TO W PANELU STEROWANIA KARTY GRAFICZNEJ"
+
+vsyncContinueMessage "WCIŚNIJ Y ABY KONTYNUOWAĆ Z WYŁĄCZONĄ SYNCHRONIZACJĄ PIONOWĄ LUB WCIŚNIJ ESC ABY WYJŚĆ"
+
+
+fpsErrorLogin "BŁĄD:  WYKRYTO NIESTABILNE FPS.  CZY ZMIENIŁY SIĘ USTAWIENIA SYNCHRONIZACJI PIONOWEJ?"
+
+
+
+loading "ŁADOWANIE"
+
+sprites "GRAFIKI"
+objects "OBIEKTY"
+transitions "TŁUMACZENIA"
+animations "ANIMACJE"
+categories "KATEGORIE"
+
+groundTextures "TEKSTURY PODŁOŻA"
+
+
+spritesRebuild "GRAFIKI##(PRZEBUDOWYWANIE CACHE)"
+objectsRebuild "OBIEKTY##(PRZEBUDOWYWANIE CACHE)"
+transitionsRebuild "TŁU,ACZENIA##(PRZEBUDOWYWANIE CACHE)"
+animationsRebuild "ANIMACJE##(PRZEBUDOWYWANIE CACHE)"
+categoriesRebuild "KATEGORIE##(PRZEBUDOWYWANIE CACHE)"
+
+
+
+
+email "EMAIL"
+atSignTip "WSTAW ZNAK '@' (INTL KEYBOARDS)"
+
+
+loginButton "ZALOGUJ"
+loginNoSaveButton "TYMCZASOWO ZALOGUJ"
+settingsButton "USTAWIENIA"
+postReviewButton "NAPISZ RECENZJĘ"
+updateReviewButton "ZAKTUALIZUJ RECENZJĘ"
+
+reviewText "TEKST RECENZJI"
+reviewName "PODPIS"
+
+recommend "POLEĆ"
+recommendYes "TAK"
+recommendNo "NIE"
+
+postButton "OPUBLIKUJ"
+postReviewTip "OPUBLIKUJ NA ONEHOURONELIFE.COM"
+
+updateButton "ZAKTUALIZUJ"
+updateReviewTip "ZAKTUALIZUJ NA ONEHOURONELIFE.COM"
+
+removeButton "USUŃ"
+removeReviewTip "USUŃ Z ONEHOURONELIFE.COM"
+
+
+copyReviewTip "SKOPIUJ TEKST RENCENZJI DO SCHOWKA"
+copySelectionTip "SKOPIUJ ZAZNACZONY TEKST DO SCHOWKA"
+
+spellCheck "SPRAWDŹ PISOWNIĘ"
+
+requestDenied "PRÓBA OPUBLIKOWANIA ODRZUCONA"
+
+reviewPosted "RECENZJA ZOSTAŁA OPUBLIKOWANA"
+reviewUpdated "RECENZJA ZOSTAŁA ZAKTUALIZOWANA"
+reviewRemoved "RECENZJA ZOSTAŁA USUNIĘTA"
+
+
+
+saveTip "ZALOGUJ I ZAPISZ DANE LOGOWANIA LOKALNIE"
+noSaveTip "NIE ZAPISUJ DANYCH LOGOWANIA LOKALNIE"
+
+
+accountKey "KLUCZ"
+paste "WKLEJ"
+copy "KOPIUJ"
+clear "WYCZYŚĆ"
+
+charLimit "LIMIT 5000 ZNAKÓW OSIĄGNIĘTY"
+
+
+
+backButton "POWRÓT"
+restartButton "RESTART"
+fullscreen "PEŁNY EKRAN:"
+musicLoudness "GŁOŚNOŚĆ MUZYKI:"
+soundLoudness "GŁOŚNOŚĆ DŹWIĘKÓW:"
+
+currentFPS "AKTUALNA ILOŚĆ KLATEK:"
+targetFPS "DOCELOWA ILOŚĆ KLATEK:"
```

Comment:
I'd go with "liczba klatek" instead of "ilość klatek".

Reply:
You are right:
https://www.youtube.com/watch?v=Gc31UQ-C6dw